core/src/integration-test/java/com/scalar/db/transaction/jdbc/JdbcTransactionIntegrationTest.java

```diff
@@ -37,7 +37,7 @@ public void abort_forOngoingTransaction_ShouldAbortCorrectly() {}
   @Override
   public void rollback_forOngoingTransaction_ShouldRollbackCorrectly() {}
 
-  @Disabled
+  @Disabled("JDBC transaction doesn't support implicit pre-read")
```

Comment:
Can we change this as follows?:

```suggestion
  @Disabled("JDBC transactions don't support implicit pre-read")
```